src/MultiReport/Mouse.cpp

```diff
@@ -67,14 +67,28 @@ static const uint8_t mouse_hid_descriptor_[] PROGMEM = {
   D_END_COLLECTION                                // END_COLLECTION
 };
 
+
+
 Mouse_::Mouse_() {
+
+  // On all architectures other than AVR, we abide by the PluggableUSB contract
+  // and plug in the constructor
+#ifndef __AVR__
+  static HIDSubDescriptor node(mouse_hid_descriptor_,
+                               sizeof(mouse_hid_descriptor_));
+  HID().AppendDescriptor(&node);
+#endif
+
 }
 
 void Mouse_::begin() {
+  // On AVR, we append the descriptor in begin, to give the compiler
+  // a chance to optimize it out and save space
+#ifdef __AVR__
   static HIDSubDescriptor node(mouse_hid_descriptor_,
                                sizeof(mouse_hid_descriptor_));
   HID().AppendDescriptor(&node);
-
+#endif
```

Comment:
I think this is unnecessary, for the same reasons as with BootKeyboard above. We should just do it in the constructor on all platforms.